frontend/src/Explorer/Types/Actions.purs

```diff
@@ -71,6 +71,8 @@ data Action
     -- clock
     | SetClock DateTime
     | UpdateClock
+    -- socket-io workaround
+    | ReloadSignal
```

Comment:
Lets not bother with this. Its disposable branch and this code probably won't live much.

If its annoying feel free to update.